src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/SearchHelper/WindowsSearchAPI.cs

```diff
@@ -91,11 +87,17 @@ public void InitQueryHelper(out ISearchQueryHelper queryHelper, int maxCount)
             queryHelper.QueryMaxResults = maxCount;
 
             // Set list of columns we want to display, getting the path presently
-            queryHelper.QuerySelectColumns = "System.ItemUrl, System.FileName, System.FileAttributes";
+            queryHelper.QuerySelectColumns = "System.ItemUrl, System.FileName";
 
             // Set additional query restriction
             queryHelper.QueryWhereRestrictions = "AND scope='file:'";
-
+            
+            if (!DisplayHiddenFiles)
```

Comment:
This only handles the case when `DisplayHiddenFiles` is false. Can you please add the other case as well ?

Reply:
When it's true it doesn't need to add any QueryWhereRestrictions (e.g. all files are allowed to show up). 
I could add a comment/empty else  case to make this more clear?